pkg/local_object_storage/metabase/util.go

```diff
@@ -97,6 +108,12 @@ func addressKey(addr *object.Address) []byte {
 	return []byte(addr.String())
 }
 
+// parses object address formed by addressKey.
+func addressFromKey(k []byte) (*object.Address, error) {
```

Comment:
Would be nice to reuse this function in:
- pkg/local_object_storage/metabase/select.go:131
- pkg/local_object_storage/metabase/select.go:439
- pkg/local_object_storage/metabase/iterators.go:134
- pkg/local_object_storage/metabase/movable.go:134

Reply:
Done